conf/gis.config

```diff
@@ -0,0 +1,27 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for GIS (Aquila)
+ * -------------------------------------------------
+ * Defines reference genomes, using iGenome paths
+ * Imported under the default 'standard' Nextflow
+ * profile in nextflow.config
+ */
+
+process {
+  executor = 'sge'
+  clusterOptions = { "-l mem_free=" + task.memory.toString().replaceAll(/[\sB]/,'') }
+  penv = 'OpenMP'
+  maxRetries = 3
+  errorStrategy = { task.attempt < 2 ? 'retry' : 'finish' }
+  // auto translate container name into conda environment name
+  beforeScript = { 'source /mnt/projects/rpd/rc/init.2017-04; module load miniconda3; set +u; source activate nfcore-rnaseq-1.5dev; set -u;' }
+}
+params {
+  saveReference = true
+  max_memory = 128.GB
```

Comment:
These `max_*` params are the same as in `base.config` so can be safely removed.

Reply:
yeah sure.

Reply:
Yup - just delete the lines, commit the change and push - the update will automatically appear on this pull request.